openmls/src/framing/public_message.rs

```diff
@@ -45,7 +45,7 @@ pub(crate) struct MembershipTag(pub(crate) Mac);
 /// ```
 #[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
 pub(crate) struct PublicMessage {
-    content: FramedContent,
+    content: FramedContentTbs,
```

Comment:
In the spec this is `FramedContent`. Why do we have a Tbs here?

Reply:
Because it carries metadata (WireFormat and ProtocolVersion) that we have to pass down the chain anyway as we unwrap the message (i.e. from MlsMessage to PublicMessage to AuthenticatedContent, etc.). We don't have to do it this way, but I thought it would be an elegant way to do it.

Reply:
I see. That's of course a little confusing, esp. with the comment.
Maybe note it in the comment why we do this and that the (de)serialization accounts for this?
Also, we call `tls_serialize` on this in `set_membership_tag` and in `verify_membership`, which appears to want the TBS. The spec is a little weird here because the tbs contains the struct, not the other way. We have to make sure that we know when we need the tbs and when the content and don't be too confusing here.

Reply:
Good point. I added a comment to the doc comments as you suggest.

It might cause some confusion, but on the other hand the change is accounted for in the serialization/deserialization functions and the fields are private, so anything outside of the module doesn't need to worry about it.

Both set_membership_tag and verify_membership should use the TBS, which I believe they indeed do.